app/controllers/home_controller.rb

```diff
@@ -5,7 +5,23 @@ def type;end
 
   def faqs; end
 
-  def profile_psychologist; end
+  def profile_psychologist; 
+    
+  end
+
+  def psychologist_List
+    @Psychologist = Psychologist.order(params[:value])
+    puts params[:value]
```

Comment:
If you don't need that puts you can erase it c:

Reply:
Agreed. You probably don't need it.